app/lib/navigation/state_file_az_question_navigation.rb

```diff
@@ -16,7 +16,7 @@ class StateFileAzQuestionNavigation
       StateFile::Questions::AzPriorLastNamesController,
       StateFile::Questions::UnemploymentController,
       StateFile::Questions::AzStateCreditsController,
-      StateFile::Questions::SubmitReturnController,
+      StateFile::Questions::AzReviewController,
```

Comment:
removed the submit return controlled since i'm now creating the submission after AzStateCreditsController